packages/http-client/src/ceramic-http-client.ts

```diff
@@ -66,11 +67,21 @@ export default class CeramicClient implements CeramicApi {
   private readonly _config: CeramicClientConfig
   public readonly _doctypeHandlers: Record<string, DoctypeHandler<Doctype>>
 
-  constructor (apiHost: string = CERAMIC_HOST, config?: CeramicClientConfig) {
-    this._config = Object.assign(DEFAULT_CLIENT_CONFIG, config ? config : {})
+  constructor (apiHost: string = CERAMIC_HOST, config: CeramicClientConfig = {}) {
+    this._config = config
+
+    // fill missing configuration from default
+    Object.keys(DEFAULT_CLIENT_CONFIG).forEach((key) => {
```

Comment:
could this be written as 
`this._config = { ...DEFAULT_CLIENT_CONFIG, ...config }`?